forch/forchestrator.py

```diff
@@ -45,7 +45,7 @@
 _FAUCET_PROM_PORT_DEFAULT = 9302
 _GAUGE_PROM_HOST = '127.0.0.1'
 _GAUGE_PROM_PORT_DEFAULT = 9303
-_CONFIG_HASH_MAX_RETRY_DEFAULT = '5'
+_CONFIG_HASH_COOLING_SEC_DEFAULT = '30'
```

Comment:
COOLING?  Not sure that's right -- timeout would be more appropriate.

Reply:
Changed to TIMEOUT.